### PR TITLE
[DBMigrator] Update db_migrator to support EdgeZoneAggregator Buffer Config for T0s

### PR DESCRIPTION
**Pull request by developfast**
Update db_migrator to support EdgeZoneAggregator Neighbor Buffer Config for T0s + unit tests

#### What I did
Updated dbmigrator to support warm upgrade to new buffer config for EdgeZoneAggregator neighbors. 

Other branch PRs: 
202012 - #2768 
202205 - #2769 
202211 - #2770 

#### How I did it
Created a new version checkpoint for the change and made appropriate changes to the ConfigDB. 

#### How to verify it
Unit tests pass

### CODE REVIEW

**Comment by developfast** [comment]
Will create new PRs for 202012, 202205 and 202211

File: scripts/db_migrator.py
Comment (neethajohn) [inline scripts/db_migrator.py:648]: Please confirm in your testing if this is really needed. If it is needed, would prefer to just change the speed alone

File: scripts/db_migrator.py
Comment (neethajohn) [inline scripts/db_migrator.py:945]: cannot bump up the version here since this seems to be the latest for 202211 https://github.com/sonic-net/sonic-utilities/blob/202211/scripts/db_migrator.py#L869
Please check with Vaibhav if he has plans to bump up the version here

File: scripts/db_migrator.py
Comment (developfast) [reply scripts/db_migrator.py:651]: Checking

File: scripts/db_migrator.py
Comment (developfast) [reply scripts/db_migrator.py:948]: Waiting on @vaibhavhd

File: scripts/db_migrator.py
Comment (developfast) [reply scripts/db_migrator.py:651]: This is needed - since the get_entry function returns the whole sub-dict (a dict inside a dict), resetting its value vs just the speed has the same effect.

Comment (neethajohn) [reply scripts/db_migrator.py:651]: Can you check if setting the speed is necessary? I would prefer to avoid setting anything in the port table

File: scripts/db_migrator.py
Comment (neethajohn) [inline scripts/db_migrator.py:646]: Move this inside for loop in 641. use self.configDB.set(self.configDB.CONFIG_DB, "CABLE_LENGTH|AZURE", intf, EDGEZONE_AGG_CABLE_LENGTH)

File: tests/db_migrator_input/config_db/sample-t0-edgezoneagg-config-same-cable-input.json
Comment (neethajohn) [inline tests/db_migrator_input/config_db/sample-t0-edgezoneagg-config-same-cable-input.json:121]: version_4_0_1

File: scripts/db_migrator.py
Comment (neethajohn) [inline scripts/db_migrator.py:886]: you still need all this. 4_0_1 is the latest on 202211

File: scripts/db_migrator.py
Comment (neethajohn) [inline scripts/db_migrator.py:935]: remove extra line. follow the single line delimiter pattern between functions

File: scripts/db_migrator.py
Comment (neethajohn) [inline scripts/db_migrator.py:945]: remove extra line here as well as many places below

**Review by neethajohn — DISMISSED** [review]
minor comments. Please wait for @vaibhavhd approval as well

File: scripts/db_migrator.py
Comment (developfast) [reply scripts/db_migrator.py:935]: Oh I am used the pre-commit checker enforcing 2 lines after functions - guess this repo doesn't have one

**Review by neethajohn — DISMISSED** [review]

File: scripts/db_migrator.py
Comment (vaibhavhd) [inline scripts/db_migrator.py:634]: It seems that you are always comparing aginst same `curr_cable_length`. Am I missing something?

File: scripts/db_migrator.py
Comment (vaibhavhd) [inline scripts/db_migrator.py:642]: What's the purpose of setting `cable_length_table` here? This update does not seem to get used anywhere hereafter.

File: scripts/db_migrator.py
Comment (developfast) [reply scripts/db_migrator.py]: We only compare the length to the first cable length found in the table - I guess the variable name may be slightly misleading - I have changed it

File: scripts/db_migrator.py
Comment (developfast) [reply scripts/db_migrator.py]: Linked in the main comment

File: scripts/db_migrator.py
Comment (neethajohn) [inline scripts/db_migrator.py:808]: Why are we updating here and many other places? This is only needed on 5_0_0. Let's discuss offline

**Review by neethajohn — DISMISSED** [review]

**Review by neethajohn — APPROVED** [review]

**Review by vaibhavhd — APPROVED** [review]

**Pull request merged**
